src/test/fuzz/banman.cpp

```diff
@@ -43,19 +43,21 @@ FUZZ_TARGET_INIT(banman, initialize_banman)
     fs::path banlist_file = gArgs.GetDataDirNet() / "fuzzed_banlist";
 
     const bool start_with_corrupted_banlist{fuzzed_data_provider.ConsumeBool()};
+    bool force_read_and_write_to_err{false};
     if (start_with_corrupted_banlist) {
         const std::string sfx{fuzzed_data_provider.ConsumeBool() ? ".dat" : ".json"};
         assert(WriteBinaryFile(banlist_file.string() + sfx,
                                fuzzed_data_provider.ConsumeRandomLengthString()));
     } else {
-        const bool force_read_and_write_to_err{fuzzed_data_provider.ConsumeBool()};
+        force_read_and_write_to_err = fuzzed_data_provider.ConsumeBool();
         if (force_read_and_write_to_err) {
             banlist_file = fs::path{"path"} / "to" / "inaccessible" / "fuzzed_banlist";
         }
     }
 
     {
-        BanMan ban_man{banlist_file, nullptr, ConsumeBanTimeOffset(fuzzed_data_provider)};
+        const auto offset{ConsumeBanTimeOffset(fuzzed_data_provider)};
```

Comment:
Unrelated to this PR, why is this called "offset"? It is actually "ban_time" - number of seconds to keep the ban entry after the misbehavior occurred.

There is some "offset" involved in `BanMan::Ban()`, but that is completely internal to `BanMan`.

Reply:
Removed `offset`